src/components/DropZone/DropZone.tsx

```diff
@@ -27,10 +27,15 @@ import {useToggle} from '../../utilities/use-toggle';
 
 import {FileUpload} from './components';
 import {DropZoneContext} from './context';
-import {fileAccepted, getDataTransferFiles} from './utils';
+import {
+  fileAccepted,
+  getDataTransferFiles,
+  defaultAllowMultiple,
+  createAllowMultipleKey,
+} from './utils';
 import styles from './DropZone.scss';
 
-type Type = 'file' | 'image';
+export type Type = 'file' | 'image';
```

Comment:
If you're going to export this I think you may need a more specific name

Reply:
I like `FileType` 
Other suggestions 🤔 
`ResourceType` - `MediaType` - `AssetType` - `DocumentType` - `UploadFileType`

Reply:
Let's go with `DropZoneFileType`